src/LongMessageService.h

```diff
@@ -41,9 +41,9 @@ class LongMessageService : public Service
 public:
 
   virtual void setController(Controller *cntrl) override { this->controller = cntrl; }
+  virtual Processed handleMessage(unsigned int opc, CANFrame *msg) override;
```

Comment:
Mark "virtual". See above. And move up so that all virtual methods are kept together.